src/main/java/org/graylog/integrations/aws/service/CloudWatchService.java

```diff
@@ -54,4 +60,32 @@ public LogGroupsResponse getLogGroupNames(String region, String awsAccessKeyId,
         LOG.debug("Log groups queried: [{}]", groupNameList);
         return LogGroupsResponse.create(groupNameList, groupNameList.size());
     }
+
+    public String addSubscriptionFilter(CloudWatchLogsClient cloudWatch, String logGroup, String streamArn,
+                                        String roleArn, String filterName,
+                                        String filterPattern) throws InterruptedException {
+        PutSubscriptionFilterRequest putSubscriptionFilterRequest =
```

Comment:
The `putSubscriptionFilterRequest` variable can be finalized. 
```suggestion
        final PutSubscriptionFilterRequest putSubscriptionFilterRequest =
```